tools/ExecuteCIStep.ps1

```diff
@@ -70,14 +73,17 @@ $ErrorActionPreference = 'Stop'
 If ($Build)
 {
     $LogFile = "$RepoArtifacts/Build.Log"
+    $buildCmdResult = "dotnet $BuildAction $RepoArtifacts/Azure.PowerShell.sln -c $Configuration -fl '/flp1:logFile=$LogFile;verbosity=quiet'"
```

Comment:
@wyunchi-ms please kindly help review this part. Is this change OK? In addition, the parameter $GenerateDocumentationFile is defined as switch. I think switch may have problem as it is always passed through. Do you think if [bool] or [string] is better?

Reply:
I think switch is OK for simple use. So I would like to use switch at here. But you can change it if you like bool better.